cmd/go-filecoin/client.go

```diff
@@ -6,7 +6,6 @@ import (
 
 	"github.com/ipfs/go-cid"
 	"github.com/ipfs/go-ipfs-cmdkit"
-	cmdkit "github.com/ipfs/go-ipfs-cmdkit"
```

Comment:
This stuff is sort of unrelated to the PR, but it needed to be fixed so my tests could run.